README.md

```diff
@@ -1,153 +1,97 @@
-# SBOM.me
+# SBOM Examples
+Welcome to SBOM-Examples. This site is a project showing off Software Bill of Material (SBOM). We focus on how to create an SBOM and more importantly, what you can do with an SBOM once you have it!
 
-Welcome to SBOM.me. This is a place to not only learn about SBOMs, but also
-start creating and using them yourself in just a couple of minutes.
+SBOM Examples will generate SBOMs from a number of popular container images every night. The list can be found [input_images/popular_dockerhub_images](here). The [https://github.com/syft](Syft) SBOM scanner is used to generate the output SBOMs. Syft is REALLY easy to run. It can output SPDX, CycloneDX, and Syft JSON SBOM files.
```

Comment:
I think these links need to be flipped `[here](input_images/popular_dockerhub_images)` and `[Syft](https://github.com/syft)`